src/RoomsSDKAdapter.js

```diff
@@ -36,6 +38,10 @@ export default class RoomsSDKAdapter extends RoomsAdapter {
 
     this.getRoomObservables = {};
     this.listenerCount = 0;
+
+    this.limit = 50;
```

Comment:
does this need to be in its own config !?

Reply:
no